storage/badger/dkg_keys.go

```diff
@@ -66,12 +66,31 @@ func (k *DKGKeys) retrieveTx(epochCounter uint64) func(tx *badger.Txn) (*dkg.DKG
 	}
 }
 
+// InsertMyDKGPrivateInfo insert the DKG private key to database when DKG was completed, and a DKG private key was successfully generated.
 func (k *DKGKeys) InsertMyDKGPrivateInfo(epochCounter uint64, info *dkg.DKGParticipantPriv) error {
 	return operation.RetryOnConflictTx(k.db, transaction.Update, k.storeTx(epochCounter, info))
 }
 
-func (k *DKGKeys) RetrieveMyDKGPrivateInfo(epochCounter uint64) (*dkg.DKGParticipantPriv, error) {
+// InsertNoDKGPrivateInfo insert a record in database to indicate that when DKG was completed,
+// the node failed DKG, there is no DKG key generated.
```

Comment:
I think there is a typo in the sentence :) Is it possible to rephrase it please?
We can also:
- use DKG fail and success, instead of completed and not completed. 
- random beacon private key or just private key, instead of DKG key.